src/features/courses/course-controller.ts

```diff
@@ -6072,6 +6072,65 @@ You can contact your student at ${options.student.email} or by replying to this
             }
         });
     }
+
+    fetchDataForExport({
```

Comment:
We should probably call this 'Archive' for consistency.

Reply:
That a good point
`CourseArchiveExport` is a good name